compiler/src/dotty/tools/dotc/core/quoted/PickledQuotes.scala

```diff
@@ -110,7 +110,8 @@ object PickledQuotes {
   /** Unpickle TASTY bytes into it's tree */
   private def unpickle(bytes: Array[Byte], splices: Seq[Any])(implicit ctx: Context): Tree = {
     val unpickler = new TastyUnpickler(bytes, splices)
-    val tree = unpickler.unpickleExpr()
+    unpickler.enter(Set(ctx.owner))
```

Comment:
Is entering ctx.owner actually useful here? Otherwise you could just do Set.empty